### PR TITLE
Fix package.json issues | Remove unused copy-webpack-plugin

### PR DESCRIPTION
**Pull request by anselmbradford**
## Changes

- Change `private` attribute in `package.json` from string to valid boolean.
- Add required `version` attribute to `package.json`.
- Remove `main` attributein `package.json` that pointed to JS file that doesn't exist.
- Move `devDependencies` next to `dependencies` in `package.json`.
- Change `homepage` attribute in `package.json` to the github pages URL, not the repo.
- Remove unused `copy-webpack-plugin`.

### CODE REVIEW

File: docs/package.json [hunk 1]
```diff
@@ -2,9 +2,19 @@
   "name": "@cfpb/cfpb-design-system",
   "version": "1.0.0",
   "description": "CFPB Design System documentation website",
-  "homepage": "https://github.com/cfpb/design-system",
+  "homepage": "https://cfpb.github.io/design-system/",
   "license": "MIT",
-  "main": "dist/main.js",
```
Comment (anselmbradford) [inline docs/package.json]: I removed this as there isn't a `dist/main.js` file. There is a `dist/js/design-system.js` file, should it be pointing to that?

File: package.json [hunk 2]
```diff
@@ -1,6 +1,7 @@
 {
   "name": "cfpb-design-system",
-  "private": "true",
+  "version": "1.0.0",
```
Comment (anselmbradford) [inline package.json]: I set this to `1.0.0` as that is the same version set in `docs/package.json`.

Comment (Scotchester) [reply package.json]: Hmm... shouldn't it be `0.0.8` to match the versions of the individual component packages?

Comment (contolini) [reply package.json]: @Scotchester that was true when the project root was also the master `capital-framework` component but we now have the master component in a [sub dir](https://github.com/cfpb/design-system/tree/master/packages/cfpb-design-system) so the version number here doesn't matter.

**Comment by contolini** [comment]
:shipit:

**Pull request merged**